src/main/java/com/aws/greengrass/mqttbridge/MQTTBridge.java

```diff
@@ -156,6 +159,8 @@ public void install() {
             return;
         }
         this.clientId = BridgeConfig.getClientId(config);
+        this.username = BridgeConfig.getUsername(config);
+        this.password = BridgeConfig.getPassword(config);
```

Comment:
We can technically disconnect and reconnect if username/password changes. Did we take a design call to reinstall in such cases?

Reply:
nope there wasn't any design reason for reinstall, and i agree that disconnect/reconnect would be more efficient.  I figured that it would be simplest to reinstall for now and then open a separate PR for disconnect/reconnect?  I started looking into this earlier and I think we'd have to do some refactor work to ensure robustness (e.g. handle exceptions from connectFuture)

Reply:
That is fine by me. Lets open a tracking ticket for that. Reinstall will cause loss of data which is not the best customer experience but I agree with taking this incrementally.